01-node-tutorial/answers/.gitignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
+.DS_Store
+writeWithPromisesAwait.js
```

Comment:
Curious why these are in your `.gitignore`. These files should actually be included in the Pull Request

Reply:
For the 'writeWithPromisesAwait.js' it is instructed to no be uploaded into github. "The file you are writing should not be sent to github, so you add the filename as another line to your .gitignore."
As for the 'writeWithPromisesThen.js' I thought maybe this one is for my own practice purposes also, so I added an entry for it in the .gitignore.

I will upload both soon.

Reply:
Ah okay, I can see how you interpreted it that way.

That is probably referring to the file you are writing/creating with the `writeFile` function instead of `writeWithPromisesAwait.js` itself.

Reply:
I see, thanks for the clarification.